notebooks/AFR-22/worksheet4.ipynb

```diff
@@ -180,6 +180,9 @@
     "    # Load the future cube\n",
     "    infile = os.path.join(CLIMDIR, gcmid + '.OND.mean.2041_2060.GERICS-REMO2015.pr.mmday-1.nc')\n",
     "    OND_future = iris.load_cube(infile)\n",
+    "    # Remove time coord from future & baseline cube to allow subtraction\n",
+    "    OND_baseline.remove_coord('time')\n",
+    "    OND_future.remove_coord('time')\n",
```

Comment:
it should be ok to just remove the coordinate from the baseline due to linient metadata. If this means the final cube has the same time coordinate as the future cube, this is desiable so that plotting works properly